### DISCUSSION TITLE
Postgresql schema

### DISCUSSION THREAD

**Discussion by DarioSantoro81** [brianluft/sqlnotebook#46] [Q&A] [answered]
Hi to all, I'm trying to setup a connection with a postgresql database. 
However I can import only tables in public schema. Is there a way to import a different schema?
Thanks, 
Dario

**Comment by electroly** [comment]
Hi Dario, it looks like this is currently impossible, and I was unable to find a workaround. I will work on adding this to the next version of SQL Notebook. I have created issue https://github.com/electroly/sqlnotebook/issues/47 to track this. Thank you for the post!

**Comment by electroly** [comment, marked as answer]
This issue should now be fixed in [version 1.2.0](https://github.com/electroly/sqlnotebook/releases/tag/v1.2.0). Please give it a try. Thanks!

**Reply by DarioSantoro81** [reply]
Thanks it's working now!
However "live database connection" works only on public schema. 
Dario

**Reply by electroly** [reply]
Aha, good catch. I didn't think that would be affected so I didn't test it -- my fault. I will take a look at this again. Thanks for trying it!
